src/dials/command_line/export.py

```diff
@@ -19,16 +19,16 @@
 
 MTZ format exports the files as an unmerged mtz file, ready for input to
 downstream programs such as Pointless and Aimless. For exporting integrated,
-but unscaled data, the required input is a models.expt file and an
+but unscaled data, the required input is an integrated.expt file and an
 integrated.refl file. For exporting scaled data, the required input is a
-models.expt file and a scaled.refl file, also passing the option
-intensity=scale.
+scald.expt file and a scaled.refl file: the intensity=scale flag will be
+derived implicitly.
```

Comment:
```suggestion
set implicitly.
```
(going for simpler language)